### ISSUE TITLE
Version 1.12.7 build failed

### ISSUE THREAD

**Issue by hantsy** [mockk/mockk#909]
I have just updated the mockk version to 1.12.7, but all Mockk APIs are not found, when I opened the jar in the locale Maven repository(~/.m2/repository), and found the mockk.jar does not contain the class files, only include a META-INF folder.

![005](https://user-images.githubusercontent.com/854520/187108003-e18d256a-24c0-4bcf-9569-99a097d0e938.png)

I have checked the online version, https://repo1.maven.org/maven2/io/mockk/mockk/1.12.7/ it also does not contains class files.

_Originally posted by @hantsy in https://github.com/mockk/mockk/discussions/908_

**Comment by aSemy** [comment]
Hi, this should be fixed when #905 is released. 

Are you using Maven? Then you can try this temp fix: https://github.com/mockk/mockk/issues/891#issuecomment-1225200585

**Comment by blueberry404** [comment]
@aSemy  I am using mockk 1.12.7 for my Junit and Android Instrumentation tests. 

Junit dependency `io.mockk:mockk:1.12.7` compiles perfectly fine and executes all the unit tests.

But Android Instrumentation test dependency `io.mockk:mockk-android:1.12.7` is failing with two errors:

1) **mockk** class is not found in the library (it's showing red).
2) Throws the error below if I try to run android tests. My min SDK version is 23. 

```
Manifest merger failed : uses-sdk:minSdkVersion 23 cannot be smaller than version 26 declared in library [io.mockk:mockk-android:1.12.7] /Users/Xyz/.gradle/caches/transforms-3/b7b0dfccead96a7770d268b1b32390cb/transformed/mockk-android-1.12.7/AndroidManifest.xml as the library might be using APIs not available in 23
	Suggestion: use a compatible library with a minSdk of at most 23,
		or increase this project's minSdk version to at least 26,
		or use tools:overrideLibrary="io.mockk" to force usage (may lead to runtime failures)

```

According to this issue, min SDK version should have been reverted to 21, but it looks like not working in my case. Please note that same android instrumentation tests compile and execute successfully with `mockk-android:1.12.5`.

**Comment by aSemy** [comment]
@blueberry404 #901 has been merged, but not released. The same for #895.

**Comment by Raibaz** [comment]
Should be fixed by v1.12.8, just released.

**Comment by stuebingerb** [comment]
With 1.12.8 we still need to depend on `mockk-jvm`
